### DISCUSSION TITLE
What dataset/ trained model did you use for facenet/arc face?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#868] [Q&A]
What dataset/ trained model did you use for facenet/arc face?

**Comment by pospielov** [comment]
We use standard models from FaceNet and InsightFace libraries

**Reply by martinenkoEduard** [reply]
Can you please clarify what models exactly.
Do you use any hand-crafted image improvements - like alignment.
Because I get much better performance in compreface than in deepface on ArcFace model.

**Reply by pospielov** [reply]
For FaceNet, it's `20180402-114759` model trained on VGGFace2
For InsightFace, it's `arcface-r100-msfdrop75` or `MobileFaceNet` models. 
We reuse the code of libraries, but I think we have some improvements. Still, as the code was written not by me, I can't say precisely what the difference is. I can point you to the python classes if you want.
